int3face/chain.json

```diff
@@ -0,0 +1,155 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "int3face",
+  "chain_type": "cosmos",
+  "chain_id": "int3face-1",
+  "pretty_name": "Int3face",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://int3face.io/",
+  "bech32_prefix": "int3",
+  "daemon_name": "int3",
+  "node_home": "$HOME/.int3faced",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uint3",
+        "fixed_min_gas_price": 0.0025,
+        "low_gas_price": 0.0025,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uint3"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/Int3facechain/bridge",
+    "recommended_version": "v0.2.7",
+    "compatible_versions": [
+      "v0.2.7"
+    ],
+    "cosmos_sdk_version": "github.com/cosmos/cosmos-sdk v0.50.8",
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.10"
+    },
+    "cosmwasm_version": "v0.50.0",
```

Comment:
Because you have cosmwasm_version and cosmwasm_enabled (which are being phased out), you'll also need the `comswasm` property with its `version` and `enable` property values matching.

Reply:
This needs to be repeated here: 
```
"cosmwasm": {
      "version": "v0.50.0",
      "enabled": true
    }
```

Reply:
Yes, actually I already have a `cosmwasm` property with its `version` and `enabled` property values.

Could I just remove the phased-out properties and use the current `cosmwasm` property only? Or should both versions be kept for backward compatibility?

Reply:
oh, yes you do. That also needs to be copied into the version object in the `versions` array.

Reply:
Sure, it is fine to omit the old properties and only include the new.

Reply:
Ok, have omitted the old properties and included the new ones